tests/model/test_instance.py

```diff
@@ -0,0 +1,97 @@
+import numpy as np
+import pytest
+from sleap_io.model.instance import (
+    Point,
+    PredictedPoint,
+    Track,
+    Instance,
+    LabeledFrame,
+    PredictedInstance,
+)
+from sleap_io.model.skeleton import Skeleton, Node
+from tests.fixture.fixtures import getDummyVideo
+
+
+def test_classes(getDummyVideo):
+
+    point_ = Point(x=0, y=0)
+    predpoint_ = PredictedPoint.from_point(point_)
+    track_ = Track()
+    skeleton_ = Skeleton.from_names(
+        nodes=["head", "thorax", "abdomen"],
+        edges=[("head", "thorax"), ("thorax", "abdomen")],
+    )
+    instance1 = Instance(skeleton=skeleton_, points={"head": point_})
+    pointsarray = np.array([[1, 1], [2, 2], [3, 3]], dtype="float32")
+    pointsconfidence = np.array([1, 2, 3], dtype="float32")
+    instance2 = Instance.from_pointsarray(points=pointsarray, skeleton=skeleton_)
+    predinstance1 = PredictedInstance(skeleton_)
+    predinstance2 = PredictedInstance.from_instance(instance1, 0.0)
+    predinstance3 = PredictedInstance.from_arrays(
+        points=pointsarray,
+        point_confidences=pointsconfidence,
+        instance_score=0.0,
+        skeleton=skeleton_,
+    )
+    predinstance4 = PredictedInstance.from_arrays(
```

Comment:
Follow python naming convention [PEP8](https://peps.python.org/pep-0008/#function-and-variable-names)
>Function names should be lowercase, with words separated by underscores as necessary to improve readability.
Variable names follow the same convention as function names.